index.js

```diff
@@ -90,6 +93,7 @@ Promise.all([globber(input), postcssrc()]).then((arr) => {
       spinner.text = `Processing ${chalk.green(`${file}`)}`
 
       postcssrc().then((config) => {
+        watcher.add(config.file)
```

Comment:
This will only add the config file to the watch list after file changes have been picked up. We should add the config file to the list of files to watch from the begining.